task_processing/plugins/stateful/stateful_executor.py

```diff
@@ -41,6 +45,10 @@ def get_event_queue(self):
     def subscribe_to_updates_for_task(self):
```

Comment:
Can we wrap the whole function to prevent the thread from crashing?

Reply:
Done.

Reply:
Dicussed with @huadongliu in person and it seems better to just catch Dynamodb exceptions.